examples/fetch_evaluations_tutorial.py

```diff
@@ -147,3 +148,28 @@ def plot_flow_compare(evaluations, top_n=10, metric='predictive_accuracy'):
 flow_names = evals.flow_name.unique()[:top_n]
 for i in range(top_n):
     print((flow_ids[i], flow_names[i]))
+
+###############################################################################
+# Obtaining evaluations with hyperparameter settings
+# ==========================================
```

Comment:
The number of equal characters needs to equal the number of characters in the header.